tensorboard/components/vz_chart_helpers/plottable-interactions.ts

```diff
@@ -216,6 +212,11 @@ class TouchDispatcher extends (Plottable.Dispatchers.Touch as any) {
 }
 
 export class PointerInteraction extends Plottable.Interactions.Pointer {
+  static preallocateTbDispatcher(component: Plottable.Component) {
+    MouseDispatcher.getDispatcher(component);
+    TouchDispatcher.getDispatcher(component);
+  }
```

Comment:
Explained offline.  It might be good to have a short comment along the lines: "allocate our custom Dispatchers first to skip Plottable's default ones".  I leave it up to you.